web/packages/teleport/src/Discover/Database/CreateDatabase/useCreateDatabase.ts

```diff
@@ -16,7 +16,7 @@
 import { useEffect, useState } from 'react';
 
 import useAttempt from 'shared/hooks/useAttemptNext';
-import { getErrMessage } from 'shared/utils/errorType';
+import { getErrMessage } from '@gravitational/shared/utils/errorType';
```

Comment:
The import should be from `shared/utils/errorType`, not `@gravitational/shared/utils/errorType`.
It should be fixed in other changed files too.

Reply:
oops auto correct